packages/cli/package.json

```diff
@@ -91,7 +91,7 @@
     "typescript": "^3.9.3",
     "url-loader": "^4.1.0",
     "webpack": "^4.41.6",
-    "webpack-dev-server": "^3.10.3",
+    "webpack-dev-server": "^3.11.0",
```

Comment:
this turned out to be useless (we were already on 3.11.0) but left it in anyway 🤷  :)